### DISCUSSION TITLE
Welcome to Optolith Discussions!

### DISCUSSION THREAD

**Discussion by elyukai** [elyukai/optolith-client#822] [General]
## 👋 Welcome!

I'm gonna use Discussions as a place to connect with you, the community. I hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with what you like most about Optolith and tell us about what you do with this community.

**Comment by Madlax85** [comment]
Is  it possible to add an option to change the Font Size and Colour ?

**Reply by elyukai** [reply]
You can change the font size with Ctrl + and - I think, but color is not adjustable.
